### PR TITLE
Updating JUCE URI on Readme

### PR DESCRIPTION
**Pull request by clebertsuconic**

### CODE REVIEW

**Comment by lpil** [comment]
Thanks!

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This repository contains tutorial content to get hackers started with the Seaboa
 
 ###JUCE Quick Start
 
-- Check out [JUCE](http://www.juce.com/about-juce) for background info
+- Check out [JUCE](http://www.juce.com/get-juce) for background info
```
Comment (lpil) [inline README.md]: Would http://www.juce.com/discover be more suitable, perhaps?

**Comment by clebertsuconic** [comment]
sure, I will amend it

**Comment by clebertsuconic** [comment]
@lpil amended

**Comment by lpil** [comment]
Thank you very much :)

**Comment by clebertsuconic** [comment]
@lpil  Yay.. my first Pull request here :) :+1: 

This is actually the very first one ! :)

Thanks

**Comment by lpil** [comment]
Pull requests always welcome :shipit:

**Pull request merged**